src/main/java/seedu/address/commons/core/AliasMapping.java

```diff
@@ -58,6 +59,14 @@ public void addAlias(Alias alias) {
         mapping.put(alias.getAliasName(), alias);
     }
 
+    /**
+     * Deletes an user-defined alias from the current mapping.
+     * @param aliasName The name of the alias to be deleted.
```

Comment:
Should there be a blank line between the description and the parameter list? I believe it's in the JavaDocs guide.

Saw this in other files too.

Reply:
Done!